admin/controllers/service-manager.js

```diff
@@ -298,6 +301,90 @@ const controller = {
     const { type, message } = result
     req.flash(type, message)
     return res.redirect('/service-manager/check-windows')
+  },
+
+  /**
+   * View sce settings.
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+  getSceSettings: async (req, res, next) => {
+    res.locals.pageTitle = 'Settings for SCE'
+    req.breadcrumbs(res.locals.pageTitle)
+    try {
+      const sceSchools = await sceService.getSceSchools()
+      res.render('service-manager/sce-settings', {
+        breadcrumbs: req.breadcrumbs(),
+        messages: res.locals.messages,
+        countriesTzData: scePresenter.getCountriesTzData(),
+        sceSchools
+      })
+    } catch (error) {
+      return next(error)
+    }
+  },
+
+  /**
+   * Add sce school form.
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+  getSceAddSchool: async (req, res, next) => {
+    req.breadcrumbs('Settings for SCE', '/service-manager/sce-settings')
+    res.locals.pageTitle = 'Add school as SCE'
+    req.breadcrumbs(res.locals.pageTitle)
+
+    try {
+      const schools = await sceService.getSchools()
+      res.render('service-manager/sce-add-school', {
+        breadcrumbs: req.breadcrumbs(),
+        err: res.error || new ValidationError(),
+        countriesTzData: scePresenter.getCountriesTzData(),
+        schools
+      })
+    } catch (error) {
+      return next(error)
+    }
+  },
+
+  /**
+   * sce school form submit handler
+   * @param req
+   * @param res
+   * @param next
+   * @returns {Promise.<void>}
+   */
+  postSceAddSchool: async (req, res, next) => {
+    const schools = await sceService.getSchools()
+    const schoolNames = schools.map(s => s.name)
+    const schoolUrns = schools.map(s => s.urn)
+
+    let validationError = await sceSchoolValidator.validate(req.body, schoolNames, schoolUrns)
```

Comment:
Minor comment: would expect the validator to be called from a service for consistency with the other controllers.